.storybook/manager-head.html

```diff
@@ -0,0 +1,103 @@
+<style type="text/css">
+    /* styles for the storybook sidebar */
+    .search-result-item {
+        border-bottom: 1px solid rgba(0, 0, 0, 0.1) !important;
+    }
+    .search-result-item--label span {
+        margin-top: 4px !important;
+    }
+    .sidebar-item,
+    .search-result-item {
+        padding-top: 0.875em !important;
+        padding-bottom: 0.875em !important;
+        font-size: 14px !important;
+        font-weight: 400 !important;
+        line-height: normal !important;
+        display: flex !important;
+        align-items: center !important;
+    }
+    .sidebar-item:first-of-type {
+        border-bottom: none !important;
+    }
+
+    /* let flex and align-items control alignment */
+    .sidebar-item > span,
+    .sidebar-item > svg {
+        margin-top: 0 !important;
+    }
+
+    /* icon size */
+    .sidebar-svg-icon,
+    .sidebar-item svg,
+    .search-result-item svg {
+        width: 4px !important;
+        height: 4px !important;
+        color: inherit !important;
+        opacity: 0 !important; /* hide icon to keep aligment with the rest of list elements */
```

Comment:
`visibility: hidden`?

Reply:
I solved the aligment by other way. Hiding icons with display none and adding some margins-right